src/twisted/test/test_twistd.py

```diff
@@ -1542,9 +1542,10 @@ def test_getLogObserverFile(self):
         """
         logFiles = _patchTextFileLogObserver(self.patch)
         filename = self.mktemp()
-        logger = app.AppLogger({"logfile": filename})
+        sut = app.AppLogger({"logfile": filename})
```

Comment:
this was shadowing the global `logger` module.
I named it `system under test` :) open to suggestions for better names